### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.mb-4)

### ISSUE THREAD

**Issue by ertascan** [ertascan/ats#3]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[İlanlar | Kariyer.net](https://ats.kariyer.net/ilanlarim/yayindakiler)
#### Element path
.mb-4
#### Snippet
`<p data-v-00a2a552="" class="mb-4"> 1 dakikada ilanınızı kolayca yayınlayabilir, 25 milyonu aşkın özgeçmiş arasından aradığınız yeteneğe en kısa sürede ulaşabilirsiniz. </p>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.84 (foreground color: #999999, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 112.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
